### ISSUE TITLE
Incorrect Validation Loss printed to terminal

### ISSUE THREAD

**Issue by CookiePPP** [NVIDIA/tacotron2#295]
https://github.com/NVIDIA/tacotron2/blob/a513db50d05ff3b2275b4e6b086b8b0c070a715f/train.py#L145
should be
`    print("Validation loss {}: {:9f}  ".format(iteration, val_loss)) `

**Comment by rafaelvalle** [comment]
Good catch! Thank you.
